### PR TITLE
magento/merchdocs#: adds items to the table in storefront settings for catalog section

### PR DESCRIPTION
**Pull request by volodymyrDashuk**
## Purpose of this pull request

This pull request (PR) adds items to the table in Storefront settings for Catalog section to the existing documentation.

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [x] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/configuration/catalog/catalog.html

### CODE REVIEW

File: src/configuration/catalog/catalog.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Cata
 |Use [Flat Catalog]({% link catalog/catalog-flat.md %}) Category|Global|Enables the flat category structure. (not recommended) Options: Yes / No|
 |Use Flat Catalog Product|Global|Enables the flat product structure. (not recommended) Options: Yes / No|
 |Swatches per Product|Store View|Determines the number of swatches available for each product. Default: `16`|
+|Show Swatches in Product List|Store View|Determines if the swatches in Product List appears. Options: Yes / No|
```
Comment (leslietilling) [inline src/configuration/catalog/catalog.md:47]:
```suggestion
|Show Swatches in Product List|Store View|Determines if the swatches appear in the Product List. Options: Yes / No|
```

File: src/configuration/catalog/catalog.md [hunk 2]
```diff
@@ -44,6 +44,8 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Cata
...
+|Show Swatch Tooltip|Store View|Determines if the swatch Tooltip appears. Options: Yes / No|
```
Comment (leslietilling) [inline src/configuration/catalog/catalog.md:48]:
```suggestion
|Show Swatch Tooltip|Store View|Determines if the swatch tooltip appears. Options: Yes / No|
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Just some minor editorial changes needed.

**Comment by volodymyrDashuk** [comment]
Hi @leslietilling 
All your suggestions have been implemented!
Could you please verify this PR?
Thank you!

File: src/configuration/catalog/catalog.md (hunk 1 above)
Comment (volodymyrDashuk) [reply src/configuration/catalog/catalog.md]: done!

File: src/configuration/catalog/catalog.md (hunk 2 above)
Comment (volodymyrDashuk) [reply src/configuration/catalog/catalog.md]: done!

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
running tests

**Pull request merged**